### DISCUSSION TITLE
[BUG]Missing XCH Coins from Chia.

### DISCUSSION THREAD

**Discussion by Wutchira** [Chia-Network/chia-blockchain#8196] [Support]
**Describe the bug**

> In the morning of 18 August 2021, I woke up and checked Chia as usual but found that 4.242002068422 coins were missing. At first, there was no trace of outgoing transfer but after my thorough examination, I learnt that the coins were transferred to the mysterious wallet address unknown to me. (I searched with https://xchscan.com/) Now, I still do not know whether the missing coins were erroneously transferred to that wallet address due to the bug or theft.>

**Questions:**

- Is there any possible way to solve this problem / get my coins back?
- Is there any possible way that I can follow-up this particular issue?

**Desktop**
- OS: Windows 10 Pro
- OS Version: 21H1
- CPU: Intel Xeon E5-2658 v3 (2 processors) Ram 64GB

**Comment by htcll411** [comment]
me too

**Comment by keliew** [comment]
Using unofficial software and poor security practise may result in theft.

It's not possible to recall those transactions...

**Comment by shaneo257** [comment]
please follow this blog post to help secure your future rewards
https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target.en.html

**Comment by diederikus** [comment]
i am missing some xch too.

**Comment by Wutchira** [comment]
@keliew I understood for your mean.  But I still wonder why that coins in wallet address do not transferring for sale or convertible coins (Common sense)

**Comment by keliew** [comment]
> @keliew I understood for your mean. But I still wonder why that coins in wallet address do not transferring for sale or convertible coins (Common sense)

Yeah, I think he should just give it to me. :D

**Comment by isipisi89** [comment]
just farm your coins in a new wallet. simple. your coins are gon byby arividerschti

**Comment by Wutchira** [comment]
@isipisi281  You mean let me stop figuring out how to recall the coins. then create a new seed?

**Comment by Wutchira** [comment]
@diederikus Where are you live. The you coins how many is missing. What do you have solve of issue.

**Comment by Wutchira** [comment]
@shaneo257  I assumed that is maybe program bug. Because the coins remains in it no movement in wallet. By common sense if theft must has outgoing transfer. Now the number of victims found is increasing day by day. (last total 26XCH)

**Comment by hoffmang9** [comment]
How many forks and which ones were you also farming - if any? Did you use your 24 words with any software or service that was not the official Chia software?

**Comment by Wutchira** [comment]
@hoffmang9 I just joined the Space Pool last month. Also never used 24 words with anything else. Now, The coin is stuck in wallet address:xch1x8yswru2ahlzce40pcuuga3yurhadg8l5elwr7uvf5rnanzea52qv09dks

I doing syncing a my wallet. after that the coins come back showing in wallet but when it sync go to the height "confirmed block 729,699" the coins is lost before synced.
![Capture](https://user-images.githubusercontent.com/89126576/130165827-1f776ef5-e8f8-4a41-8205-64921008c939.JPG)

**Comment by diederikus** [comment]
> @Wutchira  i live in netherlands i'm only missing a very small amount, somewhere in the range of 0.000035  mojos

**Comment by Wutchira** [comment]
@diederikus Even a small should not be missing. If the issue is not from user side. Mr.Bram Cohen and Chia team he should be open official contact for solve.

**Comment by keliew** [comment]
so because your wallet is not synced, you lost your coins...
why not keep it in synced?

**Comment by Wutchira** [comment]
@keliew At first my wallet was fully synced. But later I found that coins was missing. So I try to recover my coins by method deleting folder wallet on Mainnet folder. (Some detail had already told @hoffmang9) 

**The image you see is recovering.**

**Comment by Wutchira** [comment]
This is details that my coins are missing. Do you see anything wrong with pic?
<img width="1432" alt="Screen Shot 2564-08-20 at 20 06 22" src="https://user-images.githubusercontent.com/89126576/130238537-149c8a49-9ca1-447c-9853-fceffbb4b841.png">

**Comment by keliew** [comment]
Am not exactly sure, because there's no (public) identification on each wallet address...

Are you sure you aren't confused by the UTXO coin model? https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#what-is-the-coin-model

**Comment by Wutchira** [comment]
@keliew Yes, I'm sure aren't confused.

**Comment by SoMoney** [comment]
This is total BS.  Is there any way to report theft?!  
 I just checked my wallet and sure enough, it's missing 2XCH as of yesterday!
 I never post my Wallet address online.  Why is it so easy to steal CHIA?
 

Transaction Details
Buy
Exchange
Earn
Sponsored:
Chiahub.io - EU Legal XCH Pool - Join Now
Txn Hash
0xa928e7895d56f04131a08be0e5c2f9c4f179ecc6419c6d79fcd0b9654441fde2
Status
Success - Unspent
Block
825,451
Timestamp
a day ago (Sep-07-2021 12:04:45 PM) +UTC
Value
2 XCH ($436.68)
XCH Price
$255.25 / XCH
From
$MYWALLET
To
xch15xtd99q4403ymf0zqg0ykseglkjlcstq2928td3fnn536a7m9meqhkmu4k
Type
Transaction

**Reply by keliew** [reply]
So if you use any unofficial software related to Chia, and/or running an insecure system, there'll be risks of such theft.

It'd be almost no different if you use phishing websites, or plug in your credit card in a hacked terminal, for instance.

**Reply by iliaset** [reply]
I didn't report my 24 words anywhere, and I didn't join any pools. The only third - party software that was on the computer was MadMaxPlotter for Windows. However, it does not request 24 words. I can only say that a few days ago, apparently, passwords were hacked on a PC, as a result of which several mailboxes on different sites lost their passwords, and they had to be restored manually. One of the providers - Yandex-also reported DDoS attacks on their servers. I don't know if it's related, but one of the mailboxes was from them.

**Reply by PAVoutsinas** [reply]
It might just be a syncing problem. I also am missing a couple chia in my full node. In the Lite Wallet it displays the correct amount of CHia. I am actually looking for info on how to resync my wallet. Were you able to find a fix?

**Comment by iliaset** [comment]
Good afternoon!
 
I have a problem. 2 Chia coins were grown. Today, the official wallet showed a balance of "0", although there were no debits and transactions. I checked the "Chia Explorer" - similarly, the wallet balance is "0", and there are only two payment receipts in the history, and no other expenses. Where are my coins and how do I correct the balance?
 
https://chia.tt/info/address/xch10qg2hs03gdsrdhlcg49uxhfj5ummglz9j4zny67f0yyyr26ahuhqpphlnc

**Reply by keliew** [reply]
> I have a problem. 2 Chia coins were grown. Today, the official wallet showed a balance of "0", although there were no debits and transactions. I checked the "Chia Explorer" - similarly, the wallet balance is "0", and there are only two payment receipts in the history, and no other expenses. Where are my coins and how do I correct the balance?

https://xchscan.com/address/xch10qg2hs03gdsrdhlcg49uxhfj5ummglz9j4zny67f0yyyr26ahuhqpphlnc

**Reply by iliaset** [reply]
make a two-factor authorization for debiting funds in the official wallet! using an sms code to a mobile phone, for example

**Reply by keliew** [reply]
> make a two-factor authorization for debiting funds in the official wallet! using an sms code to a mobile phone, for example

Great ideas, but not a bulletproof way. Banks and financial institutions are still having to deal with fraud cases even with (strict) 2FA. Sure, it adds a layer of security, but those people would still have lost their coins/money and come to Github and post about the 'issue'. ;)

User training and education is more important, IMO.

**Reply by Wutchira** [reply]
@iliaset It's a sad story. that still can't find an answer too.

**Reply by ItsMrKnowItAll** [reply]
I had XCH but it has been gone since I haven't been able to resync my wallet, which is stuck and has been stuck. I'm still hoping for a resolution. How can I show my Chia unless my wallet syncs? I am new, but my pool info and XCH is not gone, right? I need to resolve the stuck wallet sync, nothing more. Right? I'm oversimplifying but I could also say my Chia is gone, but it's a question of sync.

**Reply by shaneo257** [reply]
your XCH remains on the blockchain, and until your client reaches the height you received it, it wont show in the client
